WebRoot/messages/ZaMsg.properties

```diff
@@ -1243,6 +1243,7 @@ NAD_MonitorHostServer = Monitor host server:
 NAD_GeneralOptions = General Options
 NAD_MailOptions = Mail Options
 NAD_MailOptionsStandardClient = Standard (HTML) client
+NAD_PopImapOptions = POP/IMAP Options
```

Comment:
Are translations not needed for all languages other than Japanese?

Reply:
Fixed.